.travis.yml

```diff
@@ -1,26 +1,74 @@
+sudo: required
+dist: trusty
+
 language: cpp
```

Comment:
Just a memorandum: It seems that Travis CI currently ignores this settings for matrix builds. https://github.com/travis-ci/travis-ci/issues/3655